packages/web-components/fast-foundation/src/text-field/text-field.spec.ts

```diff
@@ -199,6 +199,66 @@ describe("TextField", () => {
 
         await disconnect();
     });
+    it("should hide the label when start and/or end content is provided", async () => {
+        const { element, connect, disconnect } = await setup();
+        const content: HTMLImageElement = document.createElement("img");
+        content.src = "https://placehold.it/16";
+
+        const content2: HTMLImageElement = content;
+
+        await connect();
+        content.slot = "start";
+        element.appendChild(content);
+
+        expect(
+            element.shadowRoot
+                ?.querySelector("label")
+                ?.classList.contains("label__hidden")
+        ).to.be.true;
+
+        await disconnect();
+
```

Comment:
The start and end test should be split into two `it()` calls. Repetition in tests is fine, since we want to make sure all tests are isolated from each other.